chat-gpt-02-Cli-Tool/go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/atharva/chat-gpt-02
```

Comment:
Always use your own GitHub domain here. which for you will be `https://github.com/atharvaskadam` hence making the module name as `github.com/atharvaskadam/chat-gpt-02`